d2l-floating-buttons.html

```diff
@@ -105,12 +116,41 @@
 					}
 					prevDocumentHeight = documentHeight;
 				}.bind(this), 100);
+
+				if (this._checkMobileWidth()) { // move to constant?
+					var numTries = 0;
+					var initSingleRowButtons = setInterval(function() {
```

Comment:
So this is for the initial load check.  What I found was it took a variable amount of time for the container to be ready for the "shares row" check.  So I just check it every 10ms for 6 seconds (I might bring that down a bit).  Does this seem okay?  It smells a little off to me but I couldn't think of a better way.